.eslintrc.js

```diff
@@ -1,27 +1,42 @@
 module.exports = {
   extends: [
-    "eslint:recommended",
-    "plugin:import/errors",
-    "plugin:import/warnings"
+    'eslint:recommended',
+    'plugin:import/errors',
+    'plugin:import/warnings'
   ],
   env: {
     node: true,
     es6: true
   },
-  parser: "babel-eslint",
+  parser: 'babel-eslint',
   parserOptions: {
     ecmaVersion: 2016,
     sourceType: 'module'
   },
   rules: {
-    semi: ["error", "never"],
-    quotes: ["error", "single"],
+    semi: ['error', 'never'],
+    quotes: ['error', 'single'],
     indent: [2, 4],
 
-    "import/no-unresolved": [2, {commonjs: true, amd: true}],
-    "import/named": 2,
-    "import/namespace": 2,
-    "import/default": 2,
-    "import/export": 2
+    'import/no-unresolved': [2, {commonjs: true, amd: true}],
+    'import/named': 2,
+    'import/namespace': 2,
+    'import/default': 2,
+    'import/export': 2,
+
+    'no-multiple-empty-lines': [2, {'max': 1, 'maxEOF': 1}],
```

Comment:
Maybe I missed your answer,  what do you think on changing lines to 1?

Reply:
Than we would only get warnings if someone breaks that rule.